test/Dialect/FIRRTL/mem-to-reg-of-vec.mlir

```diff
@@ -1,6 +1,6 @@
 // RUN: circt-opt -pass-pipeline='firrtl.circuit(mem-to-regofvec)' %s | FileCheck  %s
 
-firrtl.circuit "Mem" {
+firrtl.circuit "Mem" attributes {annotations = [{class = "sifive.enterprise.firrtl.ConvertMemToRegOfVecAnnotation$"}]}{
```

Comment:
It would be good to add to one of these tests (or as a separate test) that the annotation is removed.

Reply:
Thanks, added the test cases.